### ISSUE TITLE
Importing 'tk' twice causes a panic

### ISSUE THREAD

**Issue by hamishforbes** [grafana/tanka#288]
If you try to import the `tk` lib more than once Tanka will panic

```
hamish@Hamish-MBP ~/tmp> tk --version
tk version v0.10.0
```

```jsonnet
local tk = import "tk";
{
    foo: tk.env.spec.apiServer,
    bar: {
        local tk = import "tk", // comment me out to fix
        baz: tk.env.spec.apiServer,
    },
}
```

```
evaluating jsonnet: INTERNAL ERROR: (CRASH) importer problem: a different instance of Contents returned when importing "<internal>/tk.libsonnet" again
goroutine 1 [running]:
runtime/debug.Stack(0xc0002dbd08, 0x12f5ac0, 0xc0001dc790)
	/usr/local/Cellar/go/1.14.2_1/libexec/src/runtime/debug/stack.go:24 +0x9d
github.com/google/go-jsonnet.(*VM).evaluateSnippet.func1(0xc0002df9d8)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/vm.go:168 +0x57
panic(0x12f5ac0, 0xc0001dc790)
	/usr/local/Cellar/go/1.14.2_1/libexec/src/runtime/panic.go:969 +0x166
github.com/google/go-jsonnet.(*importCache).importData(0xc0001732f0, 0x13761c1, 0xc, 0xc000194b0a, 0x2, 0xc0000538c0, 0x1407de0, 0xc0001e6c60, 0x1, 0x140cec0)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/imports.go:105 +0x282
github.com/google/go-jsonnet.(*importCache).importAST(0xc0001732f0, 0x13761c1, 0xc, 0xc000194b0a, 0x2, 0x203000, 0x203000, 0x0, 0x0, 0x0, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/imports.go:114 +0x74
github.com/google/go-jsonnet.(*importCache).importCode(0xc0001732f0, 0x13761c1, 0xc, 0xc000194b0a, 0x2, 0xc0000665f0, 0xc000130480, 0xc000053830, 0x8f, 0xc0001d6700, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/imports.go:155 +0x74
github.com/google/go-jsonnet.(*interpreter).evaluate(0xc0000665f0, 0x140cc20, 0xc000130480, 0x1, 0x75bcd15, 0xc0001e64b0, 0xc0001e6f00, 0x0)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:459 +0x3891
github.com/google/go-jsonnet.(*interpreter).EvalInCleanEnv(0xc0000665f0, 0xc000130500, 0x0, 0xc0001d6800, 0x140cc20, 0xc000130480, 0x0, 0x0, 0x0, 0x0, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:917 +0xbb
github.com/google/go-jsonnet.(*cachedThunk).getValue(0xc00007ed40, 0xc0000665f0, 0xc000130500, 0x0, 0x0, 0x1e1ffff, 0x400, 0x1c2f400)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/thunks.go:68 +0x8c
github.com/google/go-jsonnet.(*interpreter).evaluate(0xc0000665f0, 0x140d220, 0xc000130500, 0x0, 0x0, 0xc0001e6c60, 0x23, 0xc00000db60)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:509 +0x1478
github.com/google/go-jsonnet.(*interpreter).evaluate(0xc0000665f0, 0x140cd40, 0xc0001a5930, 0x0, 0x0, 0xc0001e6de0, 0x8e, 0xc0001d6700)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:423 +0x6a8
github.com/google/go-jsonnet.(*interpreter).evaluate(0xc0000665f0, 0x140cd40, 0xc0001a5a00, 0x0, 0xc000053880, 0xc0001e6ed0, 0x0, 0x1)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:423 +0x6a8
github.com/google/go-jsonnet.(*interpreter).evaluate(0xc0000665f0, 0x140cd40, 0xc0001a5ad0, 0x1, 0x0, 0xc0001e6f30, 0x0, 0x0)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:423 +0x6a8
github.com/google/go-jsonnet.(*interpreter).EvalInCleanEnv(0xc0000665f0, 0xc00007ee00, 0x0, 0xc0002df458, 0x140cd40, 0xc0001a5ad0, 0xc0001e4400, 0xc0001e44e0, 0x0, 0x0, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:917 +0xbb
github.com/google/go-jsonnet.(*codeUnboundField).evaluate(0xc0001dc5a0, 0xc0000665f0, 0xc00007ee00, 0x0, 0xc0001e6570, 0x0, 0xc0001e6f30, 0xc000195030, 0x3, 0xc0001e6510, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/thunks.go:92 +0x9b
github.com/google/go-jsonnet.objectIndex(0xc0000665f0, 0xc00007ee00, 0x0, 0xc0001e6570, 0x0, 0xc000195030, 0x3, 0x1407de0, 0xc0001e66c0, 0x0, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/value.go:727 +0x2a2
github.com/google/go-jsonnet.(*valueObject).index(0xc0001e6570, 0xc0000665f0, 0xc00007ee00, 0x0, 0xc000195030, 0x3, 0xc0001e6720, 0x0, 0x0, 0x0)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/value.go:500 +0x73
github.com/google/go-jsonnet.(*interpreter).manifestJSON(0xc0000665f0, 0xc00007ee00, 0x0, 0x1407de0, 0xc0001e6570, 0x0, 0xc000173140, 0x0, 0xc0001732f0)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:672 +0x54c
github.com/google/go-jsonnet.(*interpreter).manifestAndSerializeJSON(0xc0000665f0, 0xc0002df8c8, 0xc00007ee00, 0x0, 0x1407de0, 0xc0001e6570, 0xc00007ee01, 0x0, 0x0, 0x0, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:794 +0x5a
github.com/google/go-jsonnet.evaluate(0x140cf20, 0xc000171680, 0xc0001730e0, 0xc000173110, 0xc000173140, 0x1f4, 0xc0001732f0, 0x0, 0xc0001a5860, 0x30, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/interpreter.go:1236 +0x249
github.com/google/go-jsonnet.(*VM).evaluateSnippet(0xc0000665a0, 0x13761c1, 0xc, 0xc0001a8000, 0x93, 0x0, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/vm.go:177 +0x2f2
github.com/google/go-jsonnet.(*VM).EvaluateSnippet(0xc0000665a0, 0x13761c1, 0xc, 0xc0001a8000, 0x93, 0xc000143ad0, 0x100bf96, 0x18b0108, 0x17)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/google/go-jsonnet@v0.15.1-0.20200331184325-4f4aa80dd785/vm.go:194 +0x69
github.com/grafana/tanka/pkg/jsonnet.Evaluate(0xc0001a8000, 0x93, 0xc00007e940, 0x4, 0x4, 0xc000143c30, 0x1, 0x1, 0x11, 0x0, ...)
	/private/tmp/tanka-20200508-59089-ktxhqd/pkg/jsonnet/eval.go:47 +0x569
github.com/grafana/tanka/pkg/jsonnet.EvaluateFile(0xc0000149f0, 0x23, 0xc000143c30, 0x1, 0x1, 0x8f, 0x0, 0x12b3cac, 0xc000018e60)
	/private/tmp/tanka-20200508-59089-ktxhqd/pkg/jsonnet/eval.go:29 +0x1cd
github.com/grafana/tanka/pkg/tanka.evalJsonnet(0xc000018e60, 0x16, 0xc00015db90, 0xc000172930, 0xc00015db90, 0x0, 0x0)
	/private/tmp/tanka-20200508-59089-ktxhqd/pkg/tanka/parse.go:117 +0x2e4
github.com/grafana/tanka/pkg/tanka.eval(0x7ffeefbff9d0, 0x5, 0xc000172930, 0xc00007e8c0, 0x12b4b7d, 0x1325c20, 0xc0000533a0)
	/private/tmp/tanka-20200508-59089-ktxhqd/pkg/tanka/parse.go:60 +0x7d
github.com/grafana/tanka/pkg/tanka.Eval(0x7ffeefbff9d0, 0x5, 0xc000143dc0, 0x1, 0x1, 0x0, 0xc000143dd8, 0x118867a)
	/private/tmp/tanka-20200508-59089-ktxhqd/pkg/tanka/workflow.go:111 +0x68
main.evalCmd.func1(0xc000170e60, 0xc000053390, 0x1, 0x1, 0x0, 0x0)
	/private/tmp/tanka-20200508-59089-ktxhqd/cmd/tk/jsonnet.go:26 +0x8c
github.com/go-clix/cli.(*Command).execute(0xc000170e60, 0xc000053370, 0x1, 0x1, 0xc000170e60, 0xc000053370)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:118 +0x284
github.com/go-clix/cli.(*Command).Execute(0xc000114f00, 0xc000143f58, 0x4)
	/private/tmp/tanka-20200508-59089-ktxhqd/.brew_home/go/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:74 +0x145
main.main()
	/private/tmp/tanka-20200508-59089-ktxhqd/cmd/tk/main.go:60 +0x251
```

**Comment by sh0rez** [comment]
Thanks for reporting. As said on Slack, this is a direct result of our faulty implementation of `jsonnet.Importer`:

https://github.com/grafana/tanka/blob/1edbc5805dc46f8f9053db3ff582bda31f7bb836/pkg/jsonnet/importer.go#L86

Here we create a new instance of `jsonnet.Contents` all the time, but the interface expects us to return the same pointer.

**Comment by sh0rez** [comment]
Fixed in #290
